dora/core/client/fs/src/main/java/alluxio/client/file/dora/DoraCacheClient.java

```diff
@@ -177,6 +201,49 @@ protected URIStatus getStatusByGrpc(String path, GetStatusPOptions options)
     }
   }
 
+  /**
+   * Create File.
+   * @param path the file path
+   * @param options the option for creating operation
+   * @return URIStatus of new file
+   * @throws RuntimeException
+   */
+  public URIStatus createFile(String path, CreateFilePOptions options)
+      throws RuntimeException {
```

Comment:
we don't need put runtimeexception in the signature
```suggestion
      
```

Reply:
removed.